doc/api/assert.markdown

```diff
@@ -85,9 +85,27 @@ Custom error validation:
       "unexpected error"
     );
 
-## assert.doesNotThrow(block[, message])
+## assert.doesNotThrow(block[, error][, message])
 
-Expects `block` not to throw an error, see `assert.throws` for details.
+Expects `block` not to throw an error, see `assert.throws` for more details.
```

Comment:
While we're at it, can we fix the comma splice here? This should be two sentences. `...not to throw an error. See...`